#6 - Added Testimonial Widget - Chezlin Benson

chezlinbenson opened the pull request:
Created testimonial widget 

Fixes #6 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

chezlinbenson committed:
Added Testimonial Widget

Craques commented on client/src/pages/Home.css:
```diff
@@ -9,3 +9,7 @@
 	background-color: yellow;
 }
+
+.video-testimonial {
```
For css files would prefer it matched the component as well so if you have `TestimonialWidget.js`, you can then have `TestimonialWidget.css`

chezlinbenson replied:
I do have TestimonialWidget.css file, this class however applied to the section on the home page, for both widgets.

AnthonyMogotlane reviewed (commented):
Hi @chezlinbenson, 
Your CI checks are failing, please fix that.
To check a box in markdown we include an x between square brackets like so `[x]`

chezlinbenson committed:
Adjusted bootsrap card settings on widget

chezlinbenson committed:
added testimonial image to public folder

chezlinbenson committed:
Added Testimonial content and fonts

chezlinbenson committed:
Added bootsrap alignment to text

chezlinbenson committed:
Merge branch 'main' into testimonial-widget

chezlinbenson committed:
Merge branch 'main' into testimonial-widget

chezlinbenson committed:
Fixed eslint errors

AnthonyMogotlane reviewed (commented):
Great work on resolving the merge conflict and getting CI checks fixed, can you please share a screenshot of the widget.

chezlinbenson committed:
Added Styling to Testimonial and Video Widgets

chezlinbenson committed:
Fixed Lint Errors

chezlinbenson commented:
![testimonial video](https://user-images.githubusercontent.com/102852227/222982232-4d44ca71-f8f7-4056-8430-c42fd644476f.PNG)

Hi, I've updated the styling of the homepage widgets and corrected all failing eslint errors. Please can I have 2 approving reviews.
